doc/_static/extract_api_implementation.py

```diff
@@ -0,0 +1,73 @@
+import base64
+import time
+import uuid
+import json
+import requests
+
+iap_token = "integration_token"  # Use your token
+endpoint = "http://localhost:8069/iap"
+path_to_pdf = "/home/odoo/Desktop/odoo/invoice.pdf"
+version = 120  # Do not change
+
+
+def jsonrpc(path, params):
+    payload = {
+        'jsonrpc': '2.0',
+        'method': 'call',
+        'params': params,
+        'id': uuid.uuid4().hex,
+    }
+    req = requests.post(endpoint+path, json=payload, timeout=10)
+    req.raise_for_status()
+    resp = req.json()
+    return resp
+
+
+with open(path_to_pdf, "rb") as file:
+    params = {
+        'account_token': iap_token,
+        'version': 120,
+        'documents': [base64.b64encode(file.read()).decode('ascii')],
+    }
+
+response = jsonrpc("/invoice_extract/parse", params)
+print("Server received the document, status:", response['result']['status_msg'])
+
+if response['result']['status_code'] == 0:
```

Comment:
Maybe we could set the status codes that we use as global variable with meaningful names (i.e. `SUCCESS = 0` and `NOT_READY = 1`)